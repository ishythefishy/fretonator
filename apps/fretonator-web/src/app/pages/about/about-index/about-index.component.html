<main class="about__container">
  <p class="about__title">
    Hi there! <span role="img" aria-label="waving emoji">👋🏼</span>
  </p>
  <h1 class="about__subtitle">Thanks for checking out the Fretonator!</h1>
  <p class="about__copy">
    I’m a qualified music teacher that (somehow) ended up working as a lead
    software engineer (it’s a long story) and now I work in Developer Relations.
    I built this for my husband
    <span role="img" aria-label="red heart emoji">❤️</span>.
  </p>
  <p class="about__copy">
    He’s a phenomenal guitarist, and over the past few years has worked on
    learning modes and scale theory for the guitar (with a little help from me).
    Obviously, the internet is great for this kind of stuff, but what he felt
    the world wide web was lacking was an interactive and informative tool he
    could use in his practice sessions that had
    <span class="italic">everything he needed on one single page</span>.
  </p>
  <p class="about__subtitle">Enter the Fretonator!</p>
  <p class="about__copy">
    The Fretonator is your new interactive tool for learning common modes on the
    guitar, understanding the chord theory behind the modes, and most
    importantly - <span class="italic">having a jam</span>.
  </p>
  <p class="about__copy">
    The code for the Fretonator is available to
    <a
      href="https://github.com/whitep4nth3r/fretonator"
      rel="nofollow noopener noreferrer"
      target="_blank"
      class="link"
      >view on GitHub</a
    >
    and I am gratefully welcoming feedback from musicians, pull requests from
    software engineers, and anything else you are willing to contribute.
  </p>
  <p class="about__copy">
    Thank you and happy jamming!
    <span role="img" aria-label="guitar emoji">🎸</span>
  </p>
  <p class="about__signOff">Salma</p>
  <p class="about__copy">
    The Fretonator is free and always will be!
    <span role="img" aria-label="raised hands emoji">🙌🏼</span>
  </p>
</main>
