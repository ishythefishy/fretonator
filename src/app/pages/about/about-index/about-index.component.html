<main class="about__container">
  <h1 class="about__title">This is a title, you see?</h1>
  <p class="about__copy">I'm baby typewriter activated charcoal small batch pug bicycle rights fashion axe vegan squid freegan before they sold out meggings DIY tacos pabst taiyaki. Vexillologist viral cronut poutine. Photo booth jean shorts retro flannel butcher tumblr humblebrag, godard adaptogen pour-over sustainable VHS. Tousled hoodie craft beer, paleo poke pug pork belly master cleanse gochujang portland sriracha 90's lyft cloud bread. Blog farm-to-table helvetica raw denim cliche try-hard. Venmo 3 wolf moon migas kitsch flannel vice yuccie vexillologist salvia bespoke. Next level unicorn craft beer listicle, fanny pack drinking vinegar selfies quinoa readymade banjo poutine heirloom.</p>
  <p class="about__copy">IPhone heirloom aesthetic, hot chicken af vape brooklyn direct trade helvetica polaroid freegan. Lomo butcher listicle, shaman scenester slow-carb lumbersexual raclette man bun street art hoodie XOXO pinterest. Snackwave green juice blog, migas roof party unicorn selvage 8-bit pug drinking vinegar beard venmo street art cred. Retro gochujang keytar quinoa listicle four dollar toast.</p>
</main>
