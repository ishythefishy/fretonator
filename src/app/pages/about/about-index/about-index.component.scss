@import '../../../../styles/vars';
@import '../../../../styles/mixins';
@import '../../../../styles/typography';

.about__container {
  @include content_wrapper();
}

.about__title {
  @include font_large();
  margin-top: pxToRem($grid-unit * 4);
  margin-bottom: pxToRem($grid-unit * 2);
}

.about__copy {
  @include font_bodyCopy();
  margin-top: pxToRem($grid-unit * 4);
  margin-bottom: pxToRem($grid-unit * 6);
}
