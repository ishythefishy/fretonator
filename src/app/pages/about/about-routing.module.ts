import { NgModule } from '@angular/core';
import { Routes, RouterModule, Route } from '@angular/router';
import { AboutHomeComponent } from './about-home/about-home.component';

const home: Route = {
  path: '',
  pathMatch: 'full',
  component: AboutHomeComponent,
};

const routes: Routes = [home];

@NgModule({
  imports: [RouterModule.forChild(routes)],
  exports: [RouterModule],
})
export class AboutRoutingModule {}
