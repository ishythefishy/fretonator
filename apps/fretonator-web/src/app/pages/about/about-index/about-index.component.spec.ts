import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { AboutIndexComponent } from './about-index.component';
import { Component } from '@angular/core';
import { BrowserTestingModule } from '@angular/platform-browser/testing';
import { AboutModule } from '../about.module';

describe('AboutIndexComponent', () => {
  @Component({
    selector: 'app-about-index-spec',
    template: `
      <app-about-index></app-about-index>
    `
  })
  class AboutIndexComponentSpec {
  }

  let component: AboutIndexComponentSpec;
  let fixture: ComponentFixture<AboutIndexComponentSpec>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [AboutIndexComponentSpec],
      imports: [
        AboutModule,
        BrowserTestingModule
      ]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(AboutIndexComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
});
