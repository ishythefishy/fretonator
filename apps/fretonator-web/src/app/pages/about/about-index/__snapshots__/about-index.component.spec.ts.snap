// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutIndexComponent should create 1`] = `
AboutIndexComponent {
  "meta": Meta {
    "_doc": Document {
      "__zone_symbol__loadfalse": Array [
        Object {
          "runCount": 2,
          "source": "Document.addEventListener:load",
          "state": "scheduled",
          "type": "eventTask",
          "zone": "<root>",
        },
      ],
      "location": Location {
        "assign": [Function],
        "hash": "",
        "host": "localhost",
        "hostname": "localhost",
        "href": "http://localhost/",
        "origin": "http://localhost",
        "pathname": "/",
        "port": "",
        "protocol": "http:",
        "reload": [Function],
        "replace": [Function],
        "search": "",
        "toString": [Function],
      },
      Symbol(DOM SymbolTree): SymbolTreeNode {
        "cachedIndex": -1,
        "cachedIndexVersion": NaN,
        "childIndexCachedUpTo": null,
        "childrenVersion": 0,
        "firstChild": null,
        "lastChild": null,
        "nextSibling": null,
        "parent": null,
        "previousSibling": null,
      },
    },
    "_dom": BrowserDomAdapter {},
  },
  "title": Title {
    "_doc": Document {
      "__zone_symbol__loadfalse": Array [
        Object {
          "runCount": 2,
          "source": "Document.addEventListener:load",
          "state": "scheduled",
          "type": "eventTask",
          "zone": "<root>",
        },
      ],
      "location": Location {
        "assign": [Function],
        "hash": "",
        "host": "localhost",
        "hostname": "localhost",
        "href": "http://localhost/",
        "origin": "http://localhost",
        "pathname": "/",
        "port": "",
        "protocol": "http:",
        "reload": [Function],
        "replace": [Function],
        "search": "",
        "toString": [Function],
      },
      Symbol(DOM SymbolTree): SymbolTreeNode {
        "cachedIndex": -1,
        "cachedIndexVersion": NaN,
        "childIndexCachedUpTo": null,
        "childrenVersion": 0,
        "firstChild": null,
        "lastChild": null,
        "nextSibling": null,
        "parent": null,
        "previousSibling": null,
      },
    },
  },
}
`;
